packs/feat-effects/stance-rebirth-in-living-stone.json

```diff
@@ -0,0 +1,36 @@
+{
+    "_id": "zqgOjMU9TGoGwJWc",
+    "img": "icons/magic/earth/construct-stone-long-arms.webp",
+    "name": "Stance: Rebirth In Living Stone",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Rebirth In Living Stone]<br /><br /></p>\n<p>Rock overflows from within you to consume your form before cracking open to reveal your body transformed into living stone. You gain 40 temporary Hit Points. You're immune to critical hits and precision damage. You can't be pushed, pulled, or tripped while you're standing on earth or stone. Your earth Elemental Blasts deal an additional 1d10 damage of their normal type. These benefits last until the end of your next turn, but you can Sustain them up to 1 minute. After you gain temporary Hit Points from this impulse, you can't do so again for 10 minutes.</p>\n<hr />\n<p><strong>Level (20th)</strong> You gain 50 temporary Hit Points.</p>"
```

Comment:
Parts of this are doable. We can add Temp HP, the immunities, and the blast damage increase. Change the duration to 1 minute for the maximum sustain.